src/CronTask.php

```diff
@@ -1567,23 +1568,23 @@ public static function processMassiveActionsForOneItemtype(
 
         switch ($ma->getAction()) {
             case 'reset':
-                if (Config::canUpdate()) {
-                    foreach ($ids as $key) {
-                        if ($item->getFromDB($key)) {
+                foreach ($ids as $key) {
+                    if ($item->getFromDB($key)) {
+                        if (Config::canUpdate()) {
```

Comment:
Not sure I agree with that change. With original version, is `Configu::canUpdate()` was false, no query was done. With your proposal, it will always perform several queries.

Reply:
Indeed, I changed this part.